experimental/addons/process-exporter/README.md

```diff
@@ -0,0 +1,9 @@
+# process-exporter
+
+Prometheus exporter that mines /proc to report on selected processes.
+
+## Installation
+
+```shell
+vela addon enable process-exporter
+```
```

Comment:
how to use this process metrics? can you also provide a grafana dashboard or something similar?